test/persistence-hooks.suite.js

```diff
@@ -80,8 +80,10 @@ module.exports = function(dataSource, should, connectorCapabilities) {
           TestModel.create({name: 'second'}, function(err) {
             if (err) return done(err);
             var location1 = new GeoPoint({lat: 10.2, lng: 6.7});
+            var location1 = new GeoPoint({lat: 10.3, lng: 6.8});
```

Comment:
It is fixed last night by https://github.com/strongloop/loopback-datasource-juggler/pull/1160